frontend/lib/map/map/map.dart

```diff
@@ -162,7 +162,9 @@ class _MapState extends State<Map> implements MapController {
       int distSort(a, b) {
         final cA = a["geometry"]["coordinates"];
         final cB = b["geometry"]["coordinates"];
+        // ignore: argument_type_not_assignable
```

Comment:
Other solutions?

Reply:
What is the problem? Maybe you need to add type declarations to `cA` and `cB`?

Reply:
Sorry, wasn't clear here. I rather wanted to ask if there are any types that can be used because I did not find any. It would be possible to define some by ourselves if there are none.

Reply:
Done